shop/src/main/java/ro/msg/learning/controller/OrderController.java

```diff
@@ -0,0 +1,24 @@
+package ro.msg.learning.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.RequiredArgsConstructor;
+import ro.msg.learning.dto.OrderDto;
+import ro.msg.learning.entity.Order;
+import ro.msg.learning.service.interfaces.OrderService;
+
+@RestController
+@RequiredArgsConstructor
+public class OrderController {
+	private final OrderService orderService;
+
+	@PostMapping(value = "/orders")
+	@ResponseStatus(HttpStatus.CREATED)
+	public Order createOrder(@RequestBody final OrderDto orderDto) {
```

Comment:
Use DTOs for the response. If you use domain models, then the controller is tightly coupled to your domain and with any change in the domain model, you'll change the returned response. Here, for example, it would make sense to return the products and quantities ordered as well.

Reply:
Fixed